crates/order-validation/LICENSE-APACHE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Were the license files actually copied or linked? I believe so far we just linked the files.